oauthlib/openid/connect/core/grant_types/base.py

```diff
@@ -310,11 +310,15 @@ def openid_authorization_validator(self, request):
             msg = "Session user does not match client supplied user."
             raise LoginRequired(request=request, description=msg)
 
+        ui_locales = request.ui_locales if request.ui_locales else []
+        if hasattr(ui_locales, 'split'):
```

Comment:
isinstance check should be faster

Reply:
Maybe, but I'm just trying to mirror the code style already used in this file (see line 290)

Reply:
I see...